### ISSUE TITLE
it is not generating reports in all formats

### ISSUE THREAD

**Issue by RayoLG** [jrichardsz-software-architect-tools/stressify#16]
![Error archivos](https://user-images.githubusercontent.com/108030911/175170270-20dbfb41-f4f4-4203-81be-2c31e054a050.png)

For Windows it is only generating the log.txt file and the .csv file

**Comment by jrichardsz** [comment]
Only on windows 

```
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: org/apache/commons/io/output/UnsynchronizedByteArrayOutputStream
        at org.jrichardsz.tools.stressify.steps.ReportStep.createExcel(ReportStep.java:39)
```

**Comment by jrichardsz** [comment]
fixed with 1.0.5
